experiments/ClimaEarth/run_cloudy_slabplanet.jl

```diff
@@ -206,6 +206,7 @@ land_sim = bucket_init(
     tspan,
     "sphere",
     "map_static",
+    "",
```

Comment:
Could you name a variable a pass it here, instead of just passing an empty string (so that it is clear what's this argument for)